spec/requests/cloud_volumes_spec.rb

```diff
@@ -193,4 +193,20 @@
       expect(response).to have_http_status(:ok)
     end
   end
+
+  describe 'create backup' do
+    it 'it can create cloud volume backup through POST' do
+      zone = FactoryBot.create(:zone)
+      provider = FactoryBot.create(:ems_autosde, :zone => zone)
+      cloud_volume = FactoryBot.create(:cloud_volume_autosde, :ext_management_system => provider)
+
+      api_basic_authorize(action_identifier(:cloud_volumes, :create_backup, :resource_actions, :post))
+      stub_supports(cloud_volume.class, :create_backup)
+
+      payload = {:action => "create_backup", :resources => {:backup_name => "stud_backup_name"}}
+      post(api_cloud_volume_url(nil, cloud_volume), :params => payload)
+
+      expect(response).to have_http_status(:ok)
```

Comment:
Do we test the failure cases?  There was a rescue clause above, if we expect failures, we should probably test how we deal with failures such as "missing permissions/identifier" or invalid payload/request.

Reply:
yea. the original code was not checking `supports`, so that would have been caught by an unhappy path.

Reply:
Please add 2 specs.
one with a blank `api_basic_authorize` (e.g. one on line 46 w/ rejects delete) and one with `stub_supports_not` for probably a `have_http_status(:not_found)`